app/migrations/Version20180611115330.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+final class Version20180611115330 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE mooc_attachment_file ADD slug VARCHAR(255) NOT NULL, DROP uuid');
```

Comment:
The unique index constraint is missing

Reply:
This PR is in WIP. I will push soon a new commit, thx :100: :1st_place_medal: :man_cartwheeling: